### PR TITLE
test: hasValidBridge unit test

### PR DESCRIPTION
**Pull request by 0xtiti**

### CODE REVIEW

**Review by agusduha — APPROVED** [review]

File: src/utils/filteredTokenList.ts
Comment (0xArdy) [inline src/utils/filteredTokenList.ts:1]: this file should be in `data` folder

File: src/utils/filteredTokenList.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import TokenList from '@eth-optimism/tokenlist';
+import { extraTokens } from '~/data/extraTokens';
+import { TokenData } from '~/types';
+
+export const filteredTokenList: TokenData[] = [...(TokenList.tokens as TokenData[]), ...(extraTokens as TokenData[])]
```
Comment (0xArdy) [inline src/utils/filteredTokenList.ts:5]:
```suggestion
export const filteredTokenList: TokenData[] = ([...TokenList.tokens, ...extraTokens] as TokenData[])
```

Comment (0xtiti) [reply src/utils/filteredTokenList.ts]: [c166af0](https://github.com/defi-wonderland/superchain-bridge-ui/pull/125/commits/c166af09d90e1a7ffc00c5201e4aea9177158793)

**Review by 0xArdy — DISMISSED** [review]

File: src/__tests__/hasValidBridge.test.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { expect } from '@jest/globals';
+import { base, optimism, mainnet } from 'viem/chains';
+
+import { hasValidBridge } from '~/utils';
+import { filteredTokenList } from '~/data';
+
+describe('Token Bridge Validity Tests', () => {
```
[the remark this answers is not in the document]
Comment (0xArdy) [reply src/__tests__/hasValidBridge.test.ts:7]: we're gonna do it in new pr :ok_hand:

**Review by 0xArdy — APPROVED** [review]

**Review by agusduha — APPROVED** [review]

**Pull request merged**